CHANGES.md

```diff
@@ -29,9 +29,10 @@ Apollo 2.1.0
 * [fix openapi item with url illegalKey 400 error](https://github.com/apolloconfig/apollo/pull/4549)
 * [fix the exception occurred when publish/rollback namespaces with grayrelease](https://github.com/apolloconfig/apollo/pull/4564)
 * [fix create namespace with single dot 500 error](https://github.com/apolloconfig/apollo/pull/4568)
+* [Add overloaded shortcut method to register BeanDefinition](https://github.com/apolloconfig/apollo/pull/4574)
 * [Add nodejs client sdk and fix doc](https://github.com/apolloconfig/apollo/pull/4590)
 * [Move apollo-core, apollo-client, apollo-mockserver, apollo-openapi and apollo-client-config-data to apollo-java repo](https://github.com/apolloconfig/apollo/pull/4594)
 * [fix get the openapi interface that contains namespace information for deleted items](https://github.com/apolloconfig/apollo/pull/4596)
-
+* [disable spring cloud discovery when running test to speed up test process and reduce error log](https://github.com/apolloconfig/apollo/pull/4604)
```

Comment:
test's changelog doesn't need to add to CHANGES.md

Reply:
ok